### PR TITLE
chore: Revert Request types to old in X Components.

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-6299

### CODE REVIEW

**Comment by mnavarroespinosa** [comment]
There are some request types to be changed:
- `NextQueriesRequest` in `packages/x-components/src/x-modules/next-queries/store/actions/fetch-and-save-next-queries.action.ts`
- `NextQueriesRequest` in `packages/x-components/src/x-modules/next-queries/events.types.ts`
- `SearchRequest` in `packages/x-components/src/x-modules/search/store/actions/fetch-and-save-search-response.action.ts`

**Review by mnavarroespinosa — APPROVED** [review]

File: packages/x-components/src/x-modules/search/store/types.ts [hunk 1]
```diff
@@ -9,6 +8,7 @@ import {
   Redirection,
   RelatedTag,
   Result,
+  SearchResponse,
```
Comment (LuisMartinez15) [inline packages/x-components/src/x-modules/search/store/types.ts:11]: import `SearchResponse` from `@empathyco/x-adapter`

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
Good job!

**Comment by tajespasarela** [comment]
> There are some request types to be changed:
> 
> * `NextQueriesRequest` in `packages/x-components/src/x-modules/next-queries/store/actions/fetch-and-save-next-queries.action.ts`
> * `NextQueriesRequest` in `packages/x-components/src/x-modules/next-queries/events.types.ts`
> * `SearchRequest` in `packages/x-components/src/x-modules/search/store/actions/fetch-and-save-search-response.action.ts`

fixed

File: packages/x-components/src/x-modules/search/store/types.ts (hunk 1 above)
Comment (tajespasarela) [reply packages/x-components/src/x-modules/search/store/types.ts]: done

**Review by mnavarroespinosa — APPROVED** [review]

**Review by LuisMartinez15 — APPROVED** [review]
good job!

**Pull request merged**